app/authentication/authenticator.py

```diff
@@ -109,7 +109,7 @@ def load_user(extend_session: bool = True) -> Optional[User]:
         user = User(user_id, user_ik)
 
         if metadata := get_metadata(user):
-            logger.bind(tx_id=metadata.tx_id)
+            contextvars.bind_contextvars(tx_id=metadata.tx_id)
```

Comment:
Is tx_id the only var that was missing? When I compare the new output with master I notice that `request_id` is no longer in the output either

Reply:
There were few missing, all that was attached using logger.bind() in `get_questionnaire` method, we test the missing ones now in here:
```
        for output in logs.output:
            self.assertIn("questionnaire request", output)
            self.assertIn("tx_id", output)
            self.assertIn("ce_id", output)
            self.assertIn("schema_name", output)
            self.assertIn("method", output)
            self.assertIn("url_path", output)
            self.assertIn("request_id", output)
```
`type="questionnare request" is the new "questionnaire request" as contextvars's bind only accepts key-value pairs. `request_id` now added in `before_request` method and present in logs. Thanks!